draft-ietf-privacypass-architecture.md

```diff
@@ -410,15 +423,184 @@ guidelines on changes to the parametrization in {{parametrization}}.
+
+This model is useful if an Origin wants to offload attestation and issuance
+to a trusted entity. In this model, the Attester and Issuer share attestation
+context for the Client, which can be separate from the Origin's redemption context.
+
+For certain types of issuance protocols, this model separates attestation and redemption
+contexts. However, Issuance protocols that require the Issuer to learn information about
+the Origin, such as that which is described in [rate-limited], are not appropriate
+since they could link attestation and redemption contexts through the Origin name.
+
+## Joint Origin and Issuer {#deploy-joint-origin}
+
+In this model, the Origin and Issuer are operated by the same entity, separate from
+the Attester, as shown in the figure below.
+
+~~~
+                                    +--------------------------+
+      Client                        |   Issuer         Origin  |
+        |                Challenge  |                          |
+        <-----------------------------------------------+      |
+        |                           |                          |
+        |          +-----------+    |                          |
+        |          |  Attester |    |                          |
+        |          |           |    |                          |
+        |          | Attest    |    |                          |
+        +----------------->    |    |                          |
+        |          |           |    |                          |
+        |          |     TokenRequest                          |
+        +-------------------------------->                     |
+        |          |           |    |                          |
+        |          |     TokenResponse                         |
+        <--------------------------------+                     |
+        |          |           |    |                          |
+        |          +-----------+    |                          |
+        |                           |                          |
+        |                 Redeem    |                          |
+        +----------------------------------------------->      |
+                                    +--------------------------+
+~~~
+{: #fig-deploy-joint-origin title="Joint Origin and Issuer Deployment Model"}
+
+This model is useful for Origins that require Client-identifying attestation,
+e.g., through the use of application-layer account information, but do not
+otherwise want to learn information about individual Clients beyond what is
+observed during the token redemption, such as Client IP addresses.
+
+In this model, attestation and redemption contexts are separate. As a result,
+any type of attestation is suitable in this model. Moreover, any type of token
+challenge is suitable assuming Client redemption unlinkability, since no single
+party will have access to the identifying Client information and unique Origin
+information.
```

Comment:
It may be good to point out that this model is less useful if there's only one origin for the issuer — since then the attester knows exactly what the origin is. In that case, some token types aren't advisable.

Reply:
Indeed, per-origin tokens challenged by a single origin seem useless, though I'm not sure we need to explicitly call it out. Curious to hear what others think!

Reply:
I think it's important to call out, since it reduces the applicability of this deployment model. If my origin is small (only one real origin), then I should know that this probably isn't the right choice for me.